CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 
 ### Features
 
+* Added rename function for _ordination_results.py ([#2027](https://github.com/scikit-bio/scikit-bio/pull/2027)).
```

Comment:
Can be "Added `rename` method for `OrdinationResults` class."

Reply:
Sounds good! I'll fix that.